File: Assignment/Assignment09_Ced/Cedula.py
y = 1
total = 0
#y= cedula % 10
#print(y)
#cedula = cedula // 10
#print(cedula)
cedula = 175089375
while cedula > 0:
    x = cedula % 10
    
    #lista.append(x)
    
    if y == 1: 
        x = x * 2
        y = y + 1 
    else:
        x = x * 1 
        y = 1
    if x >= 10:
        z = x
        z = x % 10
        x = x // 10
        x = z + x 
    total = total + x 
    print(total)   
    #print(x)
    cedula = cedula // 10
    #print(cedula)
total = total % 10 
if total != 0:
    total = 10 - total
print("El digito verificador es: " + str(total))    
input()

File: Assignment/Assigment10_Java/Cedula.java
public class Cedula {
    public static void main(String[] args) {
        int y = 1;
        int total = 0;
        int cedula = 17562176;
        int x;
        int z;
        while (cedula > 0) {
            x = cedula % 10;
            if (y == 1) {
                x = x * 2;
                y = y + 1;

            } else {
                x = x * 1;
                y = 1;

            }
            if (x >= 10) {
                z = x;
                z = x % 10;
                x = x / 10;
                x = z + x;
                total = total + x;

                cedula = cedula / 10;
                total = total % 10;
                System.out.println(total);
            }

        }

    }
}
